text/0000-icons.md

```diff
@@ -235,10 +260,16 @@ For example, here is an icon implementation from the Classic design system befor
 
 This import will replace a direct SVG import, and the contents can be assumed to be an SVG element that is directly usable by your framework.
 
+When a developer needs an icon asset for their framework, they would only need to know to come to this package, and what it provides.
+
 # Unresolved questions
 
 >**Optional, but suggested for first drafts. What parts of the design are still TBD?**
 
 How are we sharing icons for other platforms, such as mobile native?
 
 How do we make the icons availables for designers to use?
+
+What is our process for adding new framework/format support?
```

Comment:
I think it's a matter of we should support the "big four": React, Svelte, Angular, and Vue.

Reply:
I was thinking more in terms of, when something comes up later that we need to add (or are requested to add), how do we handle that?  I think saying "we don't need to worry about that" is a valid answer, if that's the case :)